docs/pages/configuration/pipelines/functions/ensure_pull_secrets.mdx

```diff
@@ -2,3 +2,29 @@
 title: Function ensure_pull_secrets
 sidebar_label: ensure_pull_secrets
 ---
+
+This function validates existing pull secrets for a pod.
+
+## Arguments
+
+### `--pull-secret`
+
+The pull secret to be deployed.
+
+## Usage
+
+```
+ensure_pull_secrets pullSecret1 pullSecret2
+```
+
+## Flags
+
+By default, flags are disabled. If any of the flags below are enabled, the following actions are taken.
+
+### `-all`
+
+If enabled, ensures all pull secrets are deployed to a configuration.
```

Comment:
If enabled, ensures that all pull secretes are created in the target Kubernetes namespace

Reply:
fixed